<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.0.2 (Python 3.7.5 on win32)" generated="20240207 18:34:57.070" rpa="true" schemaversion="3">
<suite id="s1" name="Task" source="C:\Users\PC1\PycharmProjects\Appium\Task.robot">
<test id="s1-t1" name="Android App Automation" line="18">
<kw name="Open Chat21 Applicaiton">
<kw name="Open Application" library="AppiumLibrary">
<arg>https://127.0.0.1:4723/wd/hub</arg>
<arg>platformName=Android</arg>
<arg>deviceName=emulator-5554</arg>
<arg>appPackage=chat21.android.demo</arg>
<arg>appActivity=chat21.android.demo.SplashActivity</arg>
<arg>automationName=Uiautomator2</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<msg timestamp="20240207 18:34:57.585" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<msg timestamp="20240207 18:34:57.599" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<msg timestamp="20240207 18:34:57.617" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20240207 18:34:57.631" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20240207 18:34:57.631" endtime="20240207 18:34:57.679"/>
</kw>
<msg timestamp="20240207 18:34:57.681" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20240207 18:34:57.682" level="FAIL">MaxRetryError: HTTPSConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)')))</msg>
<status status="FAIL" starttime="20240207 18:34:57.529" endtime="20240207 18:34:57.684"/>
</kw>
<kw name="Log" library="BuiltIn">
<arg>Hi</arg>
<doc>Logs the given message with the given level.</doc>
<status status="NOT RUN" starttime="20240207 18:34:57.684" endtime="20240207 18:34:57.684"/>
</kw>
<status status="FAIL" starttime="20240207 18:34:57.529" endtime="20240207 18:34:57.684"/>
</kw>
<status status="FAIL" starttime="20240207 18:34:57.528" endtime="20240207 18:34:57.685">MaxRetryError: HTTPSConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)')))</status>
</test>
<status status="FAIL" starttime="20240207 18:34:57.092" endtime="20240207 18:34:57.688"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tasks</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Task">Task</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20240207 18:34:57.585" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<msg timestamp="20240207 18:34:57.599" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<msg timestamp="20240207 18:34:57.617" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1076)'))': /wd/hub/session</msg>
<msg timestamp="20240207 18:34:57.681" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
